### PR TITLE
Update translations.md

### PR DESCRIPTION
**Pull request by maikksmt**
Translated into German

- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/gh-pages/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

File: docs/uebersetzungen.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Übersetzungen
+
+Vielen Dank für dein Interesse an der Übersetzung der Leitfäden!
+
+## Eine Übersetzung starten
+
+Bevor du anfängst an einer Übersetzung zu arbeiten, schaue dir die [Offenen Übersetzungsanfragen](https://github.com/github/opensource.guide/labels/translation) an, um zu sehen, ob jemand anderes bereits an einer Übersetzung für deine Sprache arbeitet.
+
+Wenn nicht, dann ist heute dein Tag um diese Anstrengung zu führen! So startest du:
```
Comment (Techassi) [inline docs/uebersetzungen.md]: `(...) um diese Anstrengung zu führen` sounds a bit strange. Change it to `diese Aufgabe zu übernhemen`

File: docs/uebersetzungen.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+1. [Eine Abzweigung dieses Repository erstellen] (https://github.com/github/opensource.guide/fork)
+2. Öffne eine Anfrage mit [dieser Vorlage] (templates/translation-issue-template.md)
+3. Beginne mit der Durchsicht der Checkliste!
+
+## Eine Übersetzung aktualisieren
```
Comment (Techassi) [inline docs/uebersetzungen.md]: Change to `Eine Übersetzung regelmäßig aktualisieren`

File: docs/uebersetzungen.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Git ist ziemlich gut im Tracking von Dateien, die sich geändert haben. Wir werden versuchen es Ihnen so einfach wie möglich zu machen Ihre Übersetzung auf dem neuesten Stand zu halten.
```
Comment (Techassi) [inline docs/uebersetzungen.md]: change `geändert` to `verändert`

**Comment by Techassi** [comment]
Update file name to `translations.md`.

File: docs/uebersetzungen.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+Vielen Dank für dein Interesse an der Übersetzung der Leitfäden!
+
+## Eine Übersetzung starten
+
+Bevor du anfängst an einer Übersetzung zu arbeiten, schaue dir die [Offenen Übersetzungsanfragen](https://github.com/github/opensource.guide/labels/translation) an, um zu sehen, ob jemand anderes bereits an einer Übersetzung für deine Sprache arbeitet.
+
+Wenn nicht, dann ist heute dein Tag um diese Aufgabe zu übernehmen! So startest du:
+
+1. [Eine Abzweigung dieses Repository erstellen] (https://github.com/github/opensource.guide/fork)
```
Comment (jens1o) [inline docs/uebersetzungen.md]: Link is malformed.

File: docs/uebersetzungen.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+2. Öffne eine Anfrage mit [dieser Vorlage] (templates/translation-issue-template.md)
```
Comment (jens1o) [inline docs/uebersetzungen.md]: Link is malformed.

File: docs/uebersetzungen.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+3. Beginne mit der Durchsicht der Checkliste!
+
+## Eine Übersetzung aktuell halten
+
+Git ist ziemlich gut im Tracking von Dateien, die sich verändert haben. Wir werden versuchen es Ihnen so einfach wie möglich zu machen Ihre Übersetzung auf dem neuesten Stand zu halten.
+
+Hinweis: In diesen Anweisungen wird davon ausgegangen, dass der `Ursprung` [remote](https://git-scm.com/docs/git-remote) die Übersetzungs-Abzweigung ist. Wenn du das Repository ursprünglich nicht aus der Abzweigung geklont hast, kannst du es mit `git remote set-url origin https://github.com/ [deinname] / opensource.guide.git` aktualisieren.
```
Comment (jens1o) [inline docs/uebersetzungen.md]: Could you remove the unnecessary spaces in the github link?

File: docs/uebersetzungen.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+Hier ist die empfohlene Vorgehensweise:
+
+1. Führe `$script/sync-translation` aus, um die letzten Änderungen aus dem Upstream zusammenzuführen und eine Pull-Anforderung für Ihre Abzweigung zu öffnen.
```
Comment (jens1o) [inline docs/uebersetzungen.md]: `deine`

File: docs/translations.md [hunk 8]
```diff
@@ -8,15 +8,15 @@ Bevor du anfängst an einer Übersetzung zu arbeiten, schaue dir die [Offenen Ü
...
 
-1. [Eine Abzweigung dieses Repository erstellen] (https://github.com/github/opensource.guide/fork)
-2. Öffne eine Anfrage mit [dieser Vorlage] (templates/translation-issue-template.md)
+1. [Eine Abzweigung dieses Repository erstellen](https://github.com/github/opensource.guide/fork)
+2. Öffne eine Anfrage mit [dieser Vorlage](templates/translation-issue-template.md)
 3. Beginne mit der Durchsicht der Checkliste!
 
 ## Eine Übersetzung aktuell halten
 
 Git ist ziemlich gut im Tracking von Dateien, die sich verändert haben. Wir werden versuchen es Ihnen so einfach wie möglich zu machen Ihre Übersetzung auf dem neuesten Stand zu halten.
```
Comment (jens1o) [inline docs/translations.md]: `informal form`

File: docs/translations.md [hunk 9]
```diff
@@ -1,27 +1,27 @@
...
-1. Start working through the checklist!
+1. [Eine Abzweigung dieses Repository erstellen](https://github.com/github/opensource.guide/fork)
+1. Öffne eine Anfrage mit [dieser Vorlage](templates/translation-issue-template.md)
+1. Beginne mit der Durchsicht der Checkliste!
 
-## Keeping a translation updated
+## Eine Übersetzung aktuell halten
 
-Git is pretty good at tracking files that have changed. We'll try to make it as easy as possible for you to keep your translation up to date.
+Git ist ziemlich gut im Tracking von Dateien, die sich verändert haben. Wir werden versuchen es Ihnen so einfach wie möglich zu machen Ihre Übersetzung auf dem neuesten Stand zu halten.
```
Comment (jens1o) [inline docs/translations.md]: The informal form is still missing.

**Pull request merged**